russstation/code/modules/clothing/dwarf_clothes.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/clothing/under/dwarf
+	name = "dwarven tunic"
+	desc = "Very hip dwarven uniform."
+	icon = 'russstation/icons/obj/clothing/uniforms.dmi'
+	icon_state = "dwarf"
+	item_state = "dwarf"
+	body_parts_covered = CHEST|GROIN|ARMS
+	can_adjust = FALSE
+	species_exception = list(/datum/species/dwarf)
+	
+/obj/item/clothing/gloves/dwarf
+	name = "dwarven gloves"
+	desc = "Great for pulping people in bar fights."
+	icon = 'russstation/icons/obj/clothing/gloves.dmi'
+	icon_state = "dwarf"
+	item_state = "dwarf"
+	body_parts_covered = ARMS
+	species_exception = list(/datum/species/dwarf)
+
+/obj/item/clothing/shoes/dwarf
+	name = "dwarven shoes"
+	desc = "Standered issue dwarven mining shoes."
+	icon = 'russstation/icons/obj/clothing/shoes.dmi'

```

Comment:
you can see here that I change the icon root but there is no other icon root for when its on their character moving in four dimensions I looked extremely hard and couldn't find it

Reply:
Yeah, a lot of times the behavior will change the icon in a way that isn't defined at the top of the object definition. I'll give it a look in a near future if you can't figure it out.

Reply:
Don't worry I found it, it was hidden as fuck

Reply:
Currently just doing more icon changes